lib/logger.js

```diff
@@ -0,0 +1,13 @@
+const wrapper = (handler, name) => {
+  return function(event, context, callback, ...args) {
+    console.log(`++++++ ${name}: EVENT ++++++`);
```

Comment:
We can get `name` from `context.functionName` so it might be unnecessary to pass it a name?

Reply:
@eyko 👍 thanks